### DISCUSSION TITLE
arm64 rasbery pi 5, not worked

### DISCUSSION THREAD

**Discussion by Dmitrium12** [ORelio/Minecraft-Console-Client#2413] [Q&A] [answered]
I did everything according to the guide for android, everything worked out, but at the time of launch it gives this error (I'm from root):

-bash: ./MinecraftClient: Permission denied

If I add sudo to the command:
sudo: ./MinecraftClient: command not found

or

sudo: MinecraftClient: command not found

please help me. I will be glad of any help

**Comment by ReinforceZwei** [comment, marked as answer]
`sudo chmod a+x ./MinecraftClient` then `./MinecraftClient`
